exporter/collstats_collector.go

```diff
@@ -47,8 +49,19 @@ func (d *collstatsCollector) Collect(ch chan<- prometheus.Metric) {
 			continue
 		}
 
-		for _, m := range stats {
-			for _, metric := range buildMetrics(m) {
+		if debugStr, err := json.MarshalIndent(stats, "", "  "); err == nil {
```

Comment:
Maybe we need to check here `logrus.IsLevelEnabled(DEBUG)` to not marshal json every time even if log level is lower then DEBUG. Or anything simmilar?